source/plugins/go/src/oms.go

```diff
@@ -1753,17 +1791,40 @@ func InitializePlugin(pluginConfPath string, agentVersion string) {
 			Log("Routing container logs thru %s route...", ContainerLogsADXRoute)
 			fmt.Fprintf(os.Stdout, "Routing container logs thru %s route...\n", ContainerLogsADXRoute)
 		}
-	} else if strings.Compare(strings.ToLower(osType), "windows") != 0 { //for linux, oneagent will be default route
+	} else if IsWindows == false { //for linux, oneagent will be default route
 		ContainerLogsRouteV2 = true //default is mdsd route
 		Log("Routing container logs thru %s route...", ContainerLogsRoute)
 		fmt.Fprintf(os.Stdout, "Routing container logs thru %s route... \n", ContainerLogsRoute)
+	} else if IsAADMSIAuthMode { //for windows, check if MSI Auth mode, then send through AMA
+		ContainerLogsRouteV2 = true 
+		Log("Routing container logs thru %s route...", ContainerLogsV2Route)
+		fmt.Fprintf(os.Stdout, "Routing container logs thru %s route... \n", ContainerLogsV2Route)
 	}
 
+	ContainerLogSchemaVersion := strings.TrimSpace(strings.ToLower(os.Getenv("AZMON_CONTAINER_LOG_SCHEMA_VERSION")))
```

Comment:
naming is a little confusing. We have ContainerLogSchemaVersion, ContainerLogV2SchemaVersion. Is there a room to improve?

Reply:
Changed it to EnvContainerLogSchemaVersion. Does that make sense?